lua/treesitter-context.lua

```diff
@@ -69,6 +69,9 @@ end
 
 ---@param winid integer
 local update_single_context = throttle_by_id(function(winid)
+  -- Remove leaked contexts firstly.
+  require('treesitter-context.render').close_leaked_contexts()
```

Comment:
This function is pretty simple, I'm not sure if it worth creating separate throttler for closing leaked contexts.